examples/simple-server.rs

```diff
@@ -1,14 +1,16 @@
 extern crate simple_server;
+extern crate http;
```

Comment:
i would prefer we don't require "sibling" or "peer" deps as they are hard to discover. i think this library should re-export them.